test/main.sh

```diff
@@ -523,6 +522,8 @@ if [ "${1:-"all"}" != "snap" ] && [ "${1:-"all"}" != "cluster" ]; then
     run_test test_projects_images_volume "projects images volume"
     run_test test_projects_backups_volume "projects backups volume"
     run_test test_projects_force_delete "projects force delete"
+    run_test test_vm_empty "Empty VM"
+    run_test test_vm_pcie_bus "VM PCIe bus numbers"
```

Comment:
Something must be leaking as now the test on `lvm` fails with due to running out of space in the `server_config` test (I retried it with the same outcome). I'll let you know if I find what's going on.